documentation/blog/2025-06-17-isolated-development-environments/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Isolated Dev Environments in Goose with conainer-use"
+description: cNever worry about breaking your development setup again with containerized, git-branch-isolated development environments powered by container-use
+authors:
+    - mic
+tags: [extensions, containers, development, isolation, mcp, dagger]
+---
+
+![Dagger Logo](https://avatars.githubusercontent.com/u/78824383?v=4&s=100)
```

Comment:
Just updated our GitHub avatar to be higher resolution. If you prefer I can share the high res file directly, but this is fine

Reply:
I think this one looks ok, it is pretty small - or is there a CDN link?

Reply:
I think it's the `&s=100` making it small, without it it's fairly large.

No CDN link unfortunately, we have PNG/SVGs in a public GDrive.

Reply:
https://avatars.githubusercontent.com/u/78824383?v=4